invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/EItemDetails/components/EItemMetadata/EItemMetadata.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Segment, Container, Header, Table } from 'semantic-ui-react';
+import { openRecordEditor } from '../../../../../routes/urls';
+import { eitem as eitemApi } from '../../../../../common/api';
+import { EditButton } from '../../../components/buttons';
+
+import './EItemMetadata.scss';
+
+export default class EItemMetadata extends Component {
+  _renderUrlLinks = urls =>
+    urls.map((url, index) => (
+      <a href={url} key={url} target="_blank" rel="noopener noreferrer">
+        Link {index + 1}
+      </a>
+    ));
+  _renderHeader = () => (
+    <Grid.Row>
+      <Grid.Column width={10} verticalAlign={'middle'}>
+        <Header as="h1">EItem - {this.props.eitem.metadata.eitem_pid}</Header>
+      </Grid.Column>
+      <Grid.Column width={6} textAlign={'right'}>
+        <EditButton
+          clickHandler={() =>
+            openRecordEditor(eitemApi.url, this.props.eitem.eitem_pid)
+          }
+        />
+      </Grid.Column>
+    </Grid.Row>
+  );
+  render() {
+    const { eitem } = this.props;
+
+    return (
+      <Segment className="eitem-metadata">
+        <Grid padded columns={2}>
+          {this._renderHeader()}
+          <Grid.Row>
+            <Grid.Column>
+              <Table basic="very" definition className="metadata-table">
```

Comment:
I would recommend re-using the `backoffice/components/MetadataTable` component.

Reply:
The `backoffice/components/MetadataTable` is rendering everything in one column and in eitems we have 2 big text fields that are not fitting nice in column. But thanks for reminding that component, I had forgotten about it. Could be in the future be a bit more powerful so we can split columns in the table or create another type such as  `DoubleColumnMetadataTable`.